app/controllers/organizations_controller.rb

```diff
@@ -1,17 +1,18 @@
 class OrganizationsController < ApplicationController
   before_action :authenticate_user!
+  load_and_authorize_resource
   layout 'org_admin'
 
   def show
     @org = Organization.friendly.find(params[:id])
     session[:current_campaign] ||= Campaign.current_campaign_id(params[:id])
-    if session[:current_campaign]
```

Comment:
We actually do need this for most of the links on the Org Admin page (e.g. manually match, report on unmatched) because they relate to one campaign at a time.